styleguide/source/_patterns/05-pages/press-release.twig

```diff
@@ -1,6 +1,9 @@
 {% extends '@templates/two-column-right.twig' %}
 
 {% block pageContent %}
+  <div class="container">
+    {% include '@organisms/masthead.twig' %}
+  </div>
```

Comment:
On the `news.twig` this element is in the `header` region since it needs to span both the top of the content and the top of the `sidebarSecondary`. Putting this in the `pageContent` means that the sidebar will be inline with the top of the Masthead when the `sidebarSecondary` exists. 

I don't know that either of these approaches are correct for the reason that you pointed out yesterday that the `header` should be using the default placeholder. I think we may need an additional region at the top of the page for this full-width section of the page. Thoughts?

@designerneil do you have thoughts on this either?

Reply:
@labbydev @designerneil I'm fine with adding a new region at the top of the page (under the header) to house full-width things that need to be kept separate from header things. Something like `contentTop`?

Reply:
Sure. That makes sense to me.

Reply:
Created https://issues.ama-assn.org/browse/EWL-4536 to capture adding the new region.